apps/src/templates/projects/publishDialog/publishDialogRedux.js

```diff
@@ -6,9 +6,9 @@ import { AllPublishableProjectTypes } from '../../../util/sharedConstants';
 const SHOW_PUBLISH_DIALOG = 'publishDialog/SHOW_PUBLISH_DIALOG';
 const HIDE_PUBLISH_DIALOG = 'publishDialog/HIDE_PUBLISH_DIALOG';
 
-export const PUBLISH_REQUEST = 'shareDialog/PUBLISH_REQUEST';
-export const PUBLISH_SUCCESS = 'shareDialog/PUBLISH_SUCCESS';
-export const PUBLISH_FAILURE = 'shareDialog/PUBLISH_FAILURE';
+export const PUBLISH_REQUEST = 'publishDialog/PUBLISH_REQUEST';
+export const PUBLISH_SUCCESS = 'publishDialog/PUBLISH_SUCCESS';
+export const PUBLISH_FAILURE = 'publishDialog/PUBLISH_FAILURE';
```

Comment:
I don't actually know if I should've made this change.  I don't understand why they are inconsistent with the rest of this file. Anyone know?

Reply:
It seems like they should be consistent to me 🤷‍♀️

Reply:
+1, good cleanup!